spec/puppet-lint/plugins/check_classes/arrow_on_right_operand_line_spec.rb

```diff
@@ -11,7 +11,8 @@
 
       context 'arrow on the line of left operand' do
         let(:code) do
-          "Package['httpd']  #{operator}
+          "
+            Package['httpd']  #{operator}
```

Comment:
I would be curious whether the previous version of this spec still work? i.e. does your code also work OK when there is NOT a newline or whitespace to process?
Perhaps it would be good to leave this spec unchanged, and add an additional spec that describes itself as being 'arrow on the line of left operand, with whitespacing'